apps/src/templates/levelSummary/MultiResponses.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './summary.module.scss';
+import {Chart} from 'react-google-charts';
+import color from '@cdo/apps/util/color';
+
+const LETTERS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+const MultiResponses = ({scriptData, showCorrectAnswer = false}) => {
+  const multiAnswerCounts = (responses, answerCount) => ({
+    // Default to a count of 0 for each answer.
+    ...Object.fromEntries([...LETTERS.slice(0, answerCount)].map(l => [l, 0])),
+    // Overwrite the count for any answers that have student responses.
+    ...responses.reduce((acc, curr) => {
+      const letter = LETTERS.at(curr.text);
+      acc[letter] = acc[letter] ? acc[letter] + 1 : 1;
+      return acc;
+    }, {}),
+    /*
+    // FIXME: test data
+    ...{
+      A: 1,
+      B: 4,
+      C: 19,
+      D: 0,
+    },
+    */
+  });
+
+  const multiChartData = (data, highlightCorrect = false) => [
+    ['Answer', 'Count', {role: 'annotation'}, {role: 'style'}],
+    ...Object.entries(data).map(row => [
+      ...row,
+      row[1] +
+        (highlightCorrect && highlightCorrect.includes(row[0]) ? '✔️' : ''),
+      highlightCorrect
+        ? highlightCorrect.includes(row[0])
+          ? color.brand_primary_default
+          : color.brand_primary_light
+        : null,
+    ]),
+  ];
+
+  const answers = scriptData.level.properties.answers;
+  const answerData = multiAnswerCounts(scriptData.responses, answers.length);
+  const answerMax = Math.max(...Object.values(answerData));
+  const correctAnswers = answers.reduce((acc, cur, i) => {
+    if (cur.correct) {
+      acc.push([...LETTERS][i]);
+    }
+    return acc;
+  }, []);
+
+  // FIXME: Is this too weird?
+  const correctAnswerElement = document.getElementById(
+    'summary-correct-answer'
+  );
+  if (showCorrectAnswer) {
+    correctAnswerElement.classList.add(styles.correctAnswersContainer);
+    correctAnswerElement.classList.remove('hide');
+  } else {
+    correctAnswerElement.classList.add('hide');
+    correctAnswerElement.classList.remove(styles.correctAnswersContainer);
+  }
+
+  return (
+    <div className={styles.multiChart}>
+      {scriptData.responses.length > 0 && (
```

Comment:
do we only want to show this when there are responses? or is it worth just showing the empty chart?

Reply:
good question. I hid it because with no data, the chart was rendering in a weird way, but there might be a way to fill it with zero-data or something.

Reply:
product decision is to show the empty chart for now. i fixed it so it looks less weird.